components/ReviewCard.tsx

```diff
@@ -68,6 +81,44 @@ export const ReviewCard: React.FC<ReviewCardProps> = ({ submissionData }) => {
           </div>
           <div className="card-footer bg-white">
             {comment && <Markdown>{comment}</Markdown>}
+            <textarea
+              ref={commentTextField}
+              placeholder="Type something..."
+              style={{ width: '100%', padding: '1rem' }}
+            ></textarea>
+            <button
+              className="btn bg-success m-1 text-white"
+              onClick={async () => {
+                const textField = commentTextField.current as any
+                await accept({
+                  variables: {
+                    reviewer: Number(session.id),
+                    submissionId: submissionData.id,
+                    comment: textField.value
+                  }
+                })
+                window.location.reload()
```

Comment:
do you want to maybe just refetch the data for comments instead of reloading? maybe smoother UI

Reply:
well after you either accept/reject a submission it will disappear from the reviews page

Reply:
Instead of doing this, look at updating the cache with mutation.

Reply:
@hwong0305 updating the cache with mutation. @coltonehrman could you figure it out? It'll be the first time its done in our codebase lol.

Also, could we move this function to outside the component? Its nice to keep the components to just components, instead of being cluttered with JS logic.

Reply:
It appears that the cache is actually being updated automagically! lol I must not have noticed